documentation/src/main/asciidoc/introduction/Configuration.adoc

```diff
@@ -543,3 +543,31 @@ On the other hand, if only _some_ columns store nationalized data, use the link:
 Alternatively, you can configure SQL Server to use the UTF-8 enabled collation `_UTF8`.
 ====
 
+[[datetime-jdbc]]
+=== Date and time types and JDBC
+
+By default, Hibernate handles date and time types defined by `java.time` by:
+
+- converting `java.time` types to JDBC date/time types defined in `java.sql` when sending data to the database, and
+- reading `java.sql` types from JDBC and then converting them to `java.time` types when retrieving data from the database.
+
+This works best when the database server time zone agrees with JVM system time zone.
+
+TIP: We therefore recommend setting things up so that the database server and the JVM agree on the same time zone. **Hint:** when in doubt, UTC quite a nice time zone.
+
+There are two system configuration properties which influence this behavior:
+
+.Settings for JDBC date/time handling
+[%breakable,cols="35,~"]
+|===
+| Configuration property name           | Purpose
+
+| link:{doc-javadoc-url}org/hibernate/cfg/JdbcSettings.html#JDBC_TIME_ZONE[`hibernate.jdbc.time_zone`]          | Use an explicit time zone when interacting with JDBC
+| link:{doc-javadoc-url}org/hibernate/cfg/MappingSettings.html#JAVA_TIME_USE_DIRECT_JDBC[`hibernate.type.java_time_use_direct_jdbc`] | Read and write `java.time` types directly to and from JDBC
+|===
+
+You may set `hibernate.jdbc.time_zone` to the time zone of the database server if for some reason the JVM needs to operate in a different time zone.
+We do not recommend this approach.
+
+On the other hand, we would love to recommend the use of `hibernate.type.java_time_use_direct_jdbc`, but this option is still experimental for now, and does result in some subtle differences in behavior which might affect legacy programs using Hibernate.
```

Comment:
If, like we discussed, the difference is just in the offset of `OffsetDateTime`/`ZonedDateTime`/`OffsetTime`, and then only in cases where it's not persisted (and thus incorrect) anyway, we might want to encourage people to use this more...

Reply:
I agree but we need to make sure it's really just that.